php-classes/Slate/Courses/SectionsRequestHandler.php

```diff
@@ -110,11 +111,52 @@ public static function handleRecordRequest(ActiveRecord $Section, $action = fals
                 ]));
             case 'students':
                 return static::handleStudentsRequest($Section);
+            case false:
+                return static::handleBlogpostRequest($Section, $action);
             default:
                 return parent::handleRecordRequest($Section, $action);
         }
     }
 
+    public static function handleBlogpostRequest(Section $Section, $action)
+    {
+      $className = static::$recordClass;
+
+      $limit = isset($_REQUEST['limit']) && ctype_digit($_REQUEST['limit']) ? (integer)$_REQUEST['limit'] : 10;
+      $offset = isset($_REQUEST['offset']) && ctype_digit($_REQUEST['offset']) ? (integer)$_REQUEST['offset'] : 0;
+      $handle = $_REQUEST['blog_tag'];
```

Comment:
use `blog-tag` -- lowerhyphencase is preferred for query params by the internet at large

Reply:
@BillClinton doesn't look like this was resolved

Reply:
oh looks like you changed it in other places but not here, so I'd expect it to not be working as-is